sam_bot_description/config/bridge_config.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- ros_topic_name: "/clock"
+  gz_topic_name: "/clock"
+  ros_type_name: "rosgraph_msgs/msg/Clock"
+  gz_type_name: "gz.msgs.Clock"
+  direction: GZ_TO_ROS
+
```

Comment:
Do we need TF as well?

Reply:
I think the ekf node should publish the transforms instead of Gazebo. I have documented this [here](https://github.com/ros-navigation/docs.nav2.org/pull/618/files#diff-ec112ab174676a9646815862ab1c29c95529fa7fe6bd2b76bd8292da589b8c8aR276).

Reply:
I guess i haven't typically used EKF when using gazebo, I just let the diff drive plugin provide TF as pretty close to ground truth. I'm open to both ways, but just what I've usually done and makes it a little easier for folks to understand on a tutorial level

Reply:
I guess having EKF would be a good idea for a real robot, but not really for Gazebo. I can add the bridge and then maybe make launching EKF an argument of the launch file. Then I can edit the tutorials accordingly.

Reply:
> I guess having EKF would be a good idea for a real robot, but not really for Gazebo. I can add the bridge and then maybe make launching EKF an argument of the launch file. Then I can edit the tutorials accordingly.

Done in [7c1ddc8](https://github.com/ros-navigation/navigation2_tutorials/pull/102/commits/7c1ddc8ab2ecf3e25d7839637216bf9199a04900).